### ISSUE TITLE
Add HELP DESK aka "FAQ" to New Design open for Growing

### ISSUE THREAD

**Issue by katrinfriedmann** [fzesch/lagesonum#95]
LaGeSoNum HELP DESK 

I have a Problem with
**Pulldown** or direct klicks, to what to do next or may be it s possible to work with [typeform](http://www.typeform.com/) here? 
Security 
Authoritiez
Media 

I need Food, where should ich go?
There is Food at 1pm at house J.

> > > Questions we know the answer are answered right away.

My Number is not shown in any way since 6 weeks.

> > > taskforce@moabit-hilft.de

**Comment by fzesch** [comment]
I think we should combine established techniques:

[Ticket system](https://de.wikipedia.org/wiki/Issue-Tracking-System) for individual requests and complaints: refugees can file tickets and volunteers handle them.

[FAQ](https://de.wikipedia.org/wiki/Frequently_Asked_Questions): Collect knowledge, provide trustful information

A simple [online forum](https://de.wikipedia.org/wiki/Internetforum) is also a possibility, but usually it is hard to use on mobile devices (or does anybody know a good candidate?)

Some of our linux friend could try to set up a ticket system with support for multilingual frontends and backends.

**Comment by chrisN777** [comment]
@fzesch good idea! I would start in a different order. First FAQ, then Forum (if needed) and Mail or  ticketing system at last. We should try to help as much as we can without a real person involved. The better the FAQ the less question we get via mail.

I found a two years old FAQ with Django Example (with tests!) https://github.com/howiworkdaily/django-faq If we add something like this @katrinfriedmann can write the questions in this wonderful json syntax: https://github.com/howiworkdaily/django-faq/blob/master/faq/fixtures/faq_test_data.json
;-)

Using an existing forums like wefugee.org was another idea today. Search for lageso covers not much for now: http://www.wefugee.org/k/questions/search?utf8=%E2%9C%93&q=lageso&submitted=

If all this doesn't help let em send a mail to the taskforce. If there are too many mails and they can't be handled by the taskforce, we could still add a ticket system. For this last step typeform seems like a beautiful solution. See the incident report for example: https://admin.typeform.com/gallery/workspaces/979764

**Comment by fzesch** [comment]
@chrisN777 I see your point with providing as much information as possible through FAQ and forum. But I still think we should set up a ticket system for several reasons:
1. Documentation
2. Prepare for scalability
3. Enable people to collaborate on answers

This is all rather messy with plain e-mail. And from the refugee side there should not be much difference. But we can wait, of course with the ticket system. An important feature of the ticket system would be the availablity of an app, I think, or at least a mobile friendly interface.

**Comment by DaniGr** [comment]
I'm fully for the ticketing system!!
Type form is very nice but doesn't work on mobile devices.

**Comment by AndreGCRamos** [comment]
@fzesch I also agree the FAQ should be a priority as it is rather easy to conceive and will do the job until the other features are ready.
Also agreeing on the ticketing system as it is better for everyone involved so we should brainstorm on that tomorrow morning and go from there. I think we (the frontenders) will have enough time to do some initial concepts and prototype for this as well, if needed.

**Comment by katrinfriedmann** [comment]
@fzesch @DaniGr @AndreGCRamos Agreed! We will try to push FAQ and Ticketsystem as far as possible today!

**Comment by fzesch** [comment]
I think we can choose between [OSticket](http://osticket.com/features) and [OTRS](http://www.otrs.com/software/) ? Any preferences? @DaniGr ?

**Comment by katrinfriedmann** [comment]
@DaniGr @fzesch @ffflorian Design Ideas for FAQ & HELP DESK (as well transfered to @mcplanner and @AndreGCRamos)
![1_lagesonum_help_desk](https://cloud.githubusercontent.com/assets/15278680/10715474/2fadbb24-7b18-11e5-94a0-a1a894b96629.png)
![2_lagesonum_faq](https://cloud.githubusercontent.com/assets/15278680/10715473/2fac5e0a-7b18-11e5-8993-0f7feeb0e9f5.png)
